packages/slice-machine/tests/project/cypress/examples/1-getting-started/todo.spec.js

```diff
@@ -0,0 +1,143 @@
+/// <reference types="cypress" />
```

Comment:
I believe we can get rid of these examples

Reply:
I've removed the examples

Reply:
Sorry I missed your comment! I get your point too, but I thought it wouldn't be too difficult to start from the Cypress documentation